src/commands/subdomain.rs

```diff
@@ -89,12 +94,21 @@ pub fn subdomain(name: &str, user: &GlobalUser, project: &Project) -> Result<(),
         let res_json: Response = serde_json::from_str(&res_text)?;
         if already_has_subdomain(res_json.errors) {
             let sd = Subdomain::get(account_id, user)?;
-            msg = format!(
-                "⛔ This account already has a registered subdomain. You can only register one subdomain per account. Your subdomain is {}.workers.dev \n Status Code: {}\n Msg: {}",
-                sd,
-                res.status(),
-                res_text,
-            );
+            if sd == name {
+                println!(
+                    "{} Your subdomain {}.workers.dev has already been registered!",
```

Comment:
I would read this as "Your subdomain has already been registered (by someone else)" – is that the intent here? If not, should it be something like "You've already registered the subdomain xyz.workers.dev"?

Reply:
Yeah so this case is specifically for when the user is trying to register the subdomain that they themselves have already registered. The latter makes sense to me